src/components/UI/Text/InputText.vue

```diff
@@ -297,6 +345,48 @@ export default {
     "update:value",
     "start-icon-selected",
     "end-icon-selected",
+    "keydown",
   ],
+
+  mounted() {
+    //new instance of quilljs is created
+
+    if (this.isFormattingEnabled) {
+      this.quillEditor = new Quill(this.$refs.quillEditor, {
+        modules: {
+          toolbar: [["bold", "italic", "underline"]],
+        },
+        theme: "snow", //css for quilleditor
+        placeholder: this.placeholder, //placeholder for editor
+        formats: ["bold", "underline", "italic"], //formatting options for editor
+      });
+      this.quillEditor.root.innerHTML = this.value;
+      //invoked on input change
+      this.quillEditor.on("text-change", () => this.updateChange());
```

Comment:
The name of this method can be improved. 
The name of the method should also reflect what it does. This method emits the current value of the textbox. So try naming it similar to that